### PR TITLE
Add CVSS-based (v1, v2, v3) decision points as python classes

### LINKED ISSUE #330: Model CVSS v1, v2, v3 vector elements as decision points
Because of their ubiquitous availability in vulnerability data sets, we should provide set of decision points specifically to represent CVSS vector elements (aka _metrics_).

This would also provide us with an opportunity to further exercise our emerging decision point versioning rules, because CVSS has had more revisions over time than SSVC has.

The bulk of the work is in modeling CVSS v3 and v4, but the marginal effort to modeling CVSS v1 and v2 as well is small enough that we should just do all of them.

Edit: In order to line this up with PR #343, this issue is just going to cover v1, v2, and v3. v4 will be covered in a separate issue.

### PR DESCRIPTION
**Pull request by ahouseholder**
Depends on
- #340 
- #342 

Adds Decision Point and Decision Point Group models for CVSS v1, v2, and v3 vectors.

Includes unit tests to validate output against json schema from #340.

- Resolves #330

### CODE REVIEW

**Comment by ahouseholder** [comment]
Note: everything new in this PR that is not already in #342 is in commit 8f2bc07

**Review by j--- — CHANGES_REQUESTED** [review]
I'm flagging some things here but the general question is about how we are applying SSVC decision point versioning. 

If we accept any CVSS metric name change as a new object, we are essentially assuming that CVSS only changes names when the major/minor/patch version rules introduced in #350 don't apply. I do not believe that's the case, I think CVSS has changed metric names for messaging reasons. These are valid reasons, but they differ from our agreed SSVC decision point versioning rules. If we are properly integrating CVSS metrics as SSVC decision points, I suggest we apply our own versioning rules (which I think means we do not treat a metric name change as automatically a new object).

File: src/ssvc/decision_points/cvss/attack_complexity.py
[the remark this answers is not in the document]
Comment (ahouseholder) [reply src/ssvc/decision_points/cvss/attack_complexity.py:1]: Updated in [e3cd036](https://github.com/CERTCC/SSVC/pull/343/commits/e3cd0362343bd71cdf6f9fbcf1be48e5fc5a566f)

File: src/ssvc/decision_points/cvss/access_vector.py
[the remark this answers is not in the document]
Comment (ahouseholder) [reply src/ssvc/decision_points/cvss/access_vector.py:1]: Updated in https://github.com/CERTCC/SSVC/pull/343/commits/d54862a6d4c6654b91213ccf28833370c547d4e8

File: src/ssvc/decision_points/cvss/access_complexity.py
[the remark this answers is not in the document]
Comment (ahouseholder) [reply src/ssvc/decision_points/cvss/access_complexity.py:1]: The shortcoming of ADR 0003 is that we address versioning changes mostly in terms of changes to the underlying decision point values. We neglected to address changes to the decision point descriptors (name, key, description) itself.

**Review by j--- — APPROVED** [review]

**Pull request merged**